extensions/amp-story/0.1/test/test-amp-story.js

```diff
@@ -355,12 +354,13 @@ describes.realWin('amp-story', {
     };
     story.onResize();
     expect(isDesktopStub).to.be.calledOnce;
-    expect(addClassStub).to.be.calledWith('i-amphtml-story-landscape');
+    expect(toggleClassStub).to.be.calledWith('i-amphtml-story-landscape', true);
     story.element.style.width = '10px';
     story.element.style.height = '11px';
     story.onResize();
     expect(isDesktopStub).to.be.calledTwice;
-    expect(removeClassStub).to.be.calledOnce;
+    expect(toggleClassStub).to.be.calledWith('i-amphtml-story-landscape',
```

Comment:
why not check the computed style of the element?

Reply:
Sorry, meant the classList of the element.

Reply:
Done